### PR TITLE
Fix/refactoring

### PR DESCRIPTION
**Pull request by kncaputo**
- What is the change?
- We split saveBook function into two functions (added createHTML). We alphabetized our query selectors and event listeners.
- This is a fix/refactor.
- To review, look over entire js file
- You can test it all by visiting the deployed site.

### CODE REVIEW

**Review by Shawntru — APPROVED** [review]
This is some A+ refactoring, Kara! Splitting and creating createHTML function really helps clean up this code.

**Comment by kncaputo** [comment]
Lucy, please take a look at the refactoring Shawn and I did today after you and I spoke. Shawn and I discussed using toggle but decided using classList to add and remove classes better suited our needs. Thanks for reviewing this! @lucyconklin

File: src/main.js [hunk 1]
```diff
@@ -1,33 +1,29 @@
 // Create variables targetting the relevant DOM elements here 👇
+var coverImg = document.querySelector('.cover-image');
+var coverTitle = document.querySelector('.cover-title');
+var createNew = document.querySelector('.create-new-book-button');
 var formView = document.querySelector('.form-view');
+var homeButton = document.querySelector('.home-button');
 var homeView = document.querySelector('.home-view');
+var makeNew = document.querySelector('.make-new-button');
 var randomCov = document.querySelector('.random-cover-button');
 var saveCov = document.querySelector('.save-cover-button');
```
Comment (lucyconklin) [inline src/main.js:10]: 💯  Alphabetical order is great! We usually do this with just about everything, imports, css rules etc.

File: src/main.js [hunk 2]
```diff
@@ -77,41 +73,44 @@ function makeUserBook() {
...
-    savedCoverSection.insertAdjacentHTML('afterbegin', `
-      <div class="mini-cover" id="${currentCover.id}" ondblclick="deleteSavedBook(${currentCover.id})">
-      <img class="mini-cover" src="${currentCover.cover}">
-      <h2 class="cover-title">${currentCover.title}</h2>
-      <h3 class="tagline">A tale of ${currentCover.tagline1} and ${currentCover.tagline2}<h3>
-      <img class="price-tag" src="./assets/price.png">
-      <img class="overlay" src="./assets/overlay.png">
-      </div>
-    `);
+    createHTML();
```
Comment (lucyconklin) [inline src/main.js:76]: Much cleaner! You might consider rewriting this slightly to be something like `insertMiniCover('afterbegin', createHTML())`. And then `createHTML` would return your `htmlBlock` variable.

Not a big change, but you'll probably find yourself using this pattern a lot, and a logical part to break out is whatever constructs this html string from an object (in this case, the object is your currentCover).

File: src/main.js [hunk 3]
```diff
@@ -1,33 +1,29 @@
...
-var coverTitle = document.querySelector('.cover-title');
 var viewSaved = document.querySelector('.view-saved-button');
-var createNew = document.querySelector('.create-new-book-button');
-var tag1 = document.querySelector('.tagline-1');
-var tag2 = document.querySelector('.tagline-2');
-var savedCoverSection = document.querySelector('.saved-covers-section');
-
 // We've provided a few variables below
 var savedCovers = [
   new Cover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
```
Comment (lucyconklin) [inline src/main.js:16]: Do you need this line anymore? It looks like you're just starting off with one saved cover. You could remove it if you want to start with no saved covers.

**Review by lucyconklin — COMMENTED** [review]
Looking good! I just had a few minor suggestions here.

**Pull request merged**